fuzzbench/test_e2e_run.py

```diff
@@ -41,3 +45,12 @@ def test_db_contains_experiment_results(self):
     def test_experiment_report_is_generated(self):
         """Fake test to be implemented later."""
         assert True
+
+    def test_default_config_is_used(self):
+        """Tests whether the default config is used for end to end test."""
+        try:
+            fake_job = Job.fetch('e2e-test',
+                                 connection=redis.Redis(host='queue-server'))
+        except rq.exceptions.NoSuchJobError:
+            assert False
+        assert fake_job is not None
```

Comment:
Just being not None is not the right way to test things, why not have one job in queue and test that exact one.

Reply:
> Just being not None is not the right way to test things, why not have one job in queue and test that exact one.

Right. This looks unclear. My thoughts were, if it can fetch this job, then it means this experiment is running with the desired test-config.yaml.